src/Administration/Resources/app/administration/src/app/snippet/de-DE.json

```diff
@@ -826,6 +826,14 @@
       }
     },
     "sw-condition-generic": {
+      "cartLineItemOfType": {
```

Comment:
does this also affect other rules? maybe it needs a general solution?

Reply:
Oh, yes I just checked, apparently this also happens for other rules :see_no_evil: 
![image](https://github.com/user-attachments/assets/31903952-cffe-4183-846b-88592714db72)

This has been removed in the following commit: https://github.com/shopware/shopware/commit/cef288c940c1146ce4f34f0e7204d66a1b370f1a

@marcelbrode what was the reason to remove the linked snippets? Or was this just an oversight?

Reply:
Yes, Linked snippets are horrible to maintain for external translators via crowdin, therefore this solution won't be accepted. Linked snippets won't be re-introduced ever again.

But of course, leaving untranslated content was not intended, but I just didn't find any usages, when I removed them. In that regard: Thank you very much for caring! :)

But! I think you can close this pull request, since there's already a refactor for rule snippets in review, which can be found here: https://github.com/shopware/shopware/pulls/CR0YD

Reply:
CC: @CR0YD